content/download/index.html.haml

```diff
@@ -258,6 +258,12 @@ title: Jenkins download and deployment
   %p{:style=>'margin-bottom:1cm;margin-top:.2cm;'}
     Jenkins Reference Architecture and one-click deployment on Oracle Cloud Infrastructure
 
+%div
+  %a{:href=>'https://github.com/civo/kubernetes-marketplace/tree/master/jenkins'}
+    %img{:src=>'/images/civo.png', :width=>'320'}
+  %p{:style=>'margin-bottom:1cm;margin-top:.2cm;'}
+    Jenkins one-click deployment on Civo Kubernetes
```

Comment:
I would prefer to include a link to civo.com somewhere, either as the link for [Civo Kubernetes](https://www.civo.com/) or as the link for the logo with the GitHub link being in the text
```suggestion
    Jenkins one-click deployment on
      %a.item{:href=>'https://www.civo.com/'}
      Civo Kubernetes
```